decentraland/unity-explorer
# fix: giant avatars

## What does this PR change?

Fixes #781 
Fixes #788 

## How to test the changes?

It may be best to test this as part of the playtesting since its a rare occurance single player. However we should ensure no obvious issues happen.

1. Play with many avatars
2. Teleport around
3. Notice no giant avatars/scaled avatars

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
